pages/topics/permissions.mdx

```diff
@@ -26,49 +30,56 @@ Below is a table of all current permissions, their integer values in hexadecimal
+| MANAGE_WEBHOOKS \*                     | `0x0000000020000000` `(1 << 29)` | Allows management and editing of webhooks                                                                                                           | T, V         |
+| MANAGE_EXPRESSIONS \*                  | `0x0000000040000000` `(1 << 30)` | Allows management and editing of emojis, stickers and soundboard sounds                                                                             |              |
+| USE_APPLICATION_COMMANDS               | `0x0000000080000000` `(1 << 31)` | Allows members to use application commands, including slash commands and context menu commands.                                                     | T, V         |
+| REQUEST_TO_SPEAK                       | `0x0000000100000000` `(1 << 32)` | Allows for requesting to speak in stage channels.                                                                                                   | S            |
+| MANAGE_EVENTS                          | `0x0000000200000000` `(1 << 33)` | Allows for creating, editing, and deleting scheduled events                                                                                         | V, S         |
+| MANAGE_THREADS \*                      | `0x0000000400000000` `(1 << 34)` | Allows for deleting and archiving threads, and viewing all private threads                                                                          | T            |
+| CREATE_PUBLIC_THREADS                  | `0x0000000800000000` `(1 << 35)` | Allows for creating public and announcement threads                                                                                                 | T            |
+| CREATE_PRIVATE_THREADS                 | `0x0000001000000000` `(1 << 36)` | Allows for creating private threads                                                                                                                 | T            |
+| USE_EXTERNAL_STICKERS                  | `0x0000002000000000` `(1 << 37)` | Allows the usage of custom stickers from other servers                                                                                              | T, V         |
+| SEND_MESSAGES_IN_THREADS               | `0x0000004000000000` `(1 << 38)` | Allows for sending messages in threads                                                                                                              | T            |
+| USE_EMBEDDED_ACTIVITIES                | `0x0000008000000000` `(1 << 39)` | Allows for using Activities (applications with the `EMBEDDED` flag) in a voice channel                                                              | V            |
+| MODERATE_MEMBERS \*\*                  | `0x0000010000000000` `(1 << 40)` | Allows for timing out users to prevent them from sending or reacting to messages in chat and threads, and from speaking in voice and stage channels |              |
```

Comment:
Same here.

```suggestion
| MODERATE_MEMBERS\*\*                   | `0x0000010000000000` `(1 << 40)` | Allows for timing out users to prevent them from sending or reacting to messages in chat and threads, and from speaking in voice and stage channels |              |
```